internal/cli/atlas/metrics/databases/describe_test.go

```diff
@@ -37,14 +37,12 @@ func TestDatabasesDescribeOpts_Run(t *testing.T) {
 		store: mockStore,
 	}
 
-	opts := listOpts.NewProcessMetricsListOptions()
-	expected := &mongodbatlas.ProcessDatabaseMeasurements{
-		ProcessMeasurements: &mongodbatlas.ProcessMeasurements{
-			Measurements: []*mongodbatlas.Measurements{},
-		},
+	params := listOpts.NewDatabaseMeasurementsAPIParams("", "hard-00-00.mongodb.net:27017", listOpts.name)
```

Comment:
[q] Why are you passing an empty element?

Reply:
It was passing an empty projectId since it wasn't populated in the listOpts previously, but I've included it in both now

Reply:
It was passing an empty projectId since it wasn't populated in the listOpts previously, but I've included it in both now